src/test-support/intercept.ts

```diff
@@ -44,6 +48,7 @@ export const mocker = (env: (key: AppEnvKeys, d?: string) => string, cy: Server,
     // changing to `/`
     path = path === '*' ? '/' : path
     const baseUrl = env('KUMA_API_URL')
+    const client = get(TOKENS.client)
```

Comment:
Again in the "much spoken about soon to come PR" this is likely to be injected properly instead of using `get`

Reply:
Ah not done this yet, but will probably do in the follow up rather than here.